hudi-cli/src/main/java/org/apache/hudi/cli/commands/BootstrapCommand.java

```diff
@@ -24,7 +24,7 @@
 import org.apache.hudi.cli.commands.SparkMain.SparkCommand;
 import org.apache.hudi.cli.utils.InputStreamConsumer;
 import org.apache.hudi.cli.utils.SparkUtil;
-import org.apache.hudi.common.bootstrap.index.BootstrapIndex;
+import org.apache.hudi.io.bootstrap.index.BootstrapIndex;
```

Comment:
I think we can keep existing bootstrap index classes under `org.apache.hudi.common.bootstrap.index`, except for `HFileBootstrapIndex` and other HFile bootstrap index related classes, to be relocated to `org.apache.hudi.common.bootstrap.index.hfile`.  And keep `HFileBootstrapIndex` containing only the comparator class under `org.apache.hudi.common.bootstrap.index` in `hudi-hadoop-common` module:
```
public class HFileBootstrapIndex {
  /**
   * This class is explicitly used as Key Comparator to workaround hard coded
   * legacy format class names inside HBase. Otherwise we will face issues with shading.
   */
  public static class HoodieKVComparator extends CellComparatorImpl {}
}
```